chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -628,8 +627,8 @@ public static DataSet integrateFunction(final DataSet function, final double xMi
         double sign = 1.0;
 
         if (Double.isFinite(xMin) && Double.isFinite(xMax)) {
-            xMinLocal = Math.min(xMin, xMax);
-            xMaxLocal = Math.max(xMin, xMax);
+            xMinLocal = java.lang.Math.min(xMin, xMax);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.min' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)